jme3-dark-laf/src/org/jme3/netbeans/plaf/darkmonkey/DarkMonkeyLookAndFeel.java

```diff
@@ -395,4 +401,29 @@ public String getThemeProperty(String name) {
     public String getDefaultThemeProperty(String name, String defaultValue) {
         return propertiesMap.get("blue").getProperty(name, defaultValue);
     }
+
+    private void applyDejaVuFontIfSupported(NimRODTheme theme) {
+        if (!userLangIsCJK()) {
+            theme.setFont(DarkMonkeyLookAndFeel.DEJA_VU_SANS_FONT);
+        }
+    }
+
+    /**
+     * DejaVu doesn't support CJK glyphs
+     * @return true if user locale is "ja" or "ko" or "zh"
+     */
+    private Boolean userLangIsCJK() {
```

Comment:
How is this for glyphs from thailand or india, I guess they would have similar problems?

Reply:
Maybe so.
If so, I started to think it's better to find the list of what language is supported or not.
I will research what language is curretly supported by DejaVu font.

Reply:
I found list of supported language code.
https://github.com/dejavu-fonts/dejavu-fonts/blob/version_2_34/dejavu-fonts/langcover.txt

Reply:
I have created a list with property file with using this langcover.txt.